test/contrib/einstein/test_steinvi.py

```diff
@@ -234,35 +232,6 @@ def model(obs):
                 assert_array_approx_equal(init_value, np.full(expected_shape, 0.0))
 
 
-def test_svgd_loss_and_grads():
```

Comment:
Stein reports the 2-norm of Stein force instead of  ELBO loss--the system converges when the norm is zero. Note that the norm may not reach zero due to stochasticity in the gradient.